MekHQ/src/mekhq/gui/view/PersonViewPanel.java

```diff
@@ -1449,7 +1449,9 @@ private JPanel fillSkills() {
                 IOption option = i.nextElement();
                 if (option.booleanValue()) {
                     JLabel lblAbility2 = new JLabel(Utilities.getOptionDisplayName(option));
-                    lblAbility2.setToolTipText(option.getDescription());
+                    lblAbility2.setToolTipText(wordWrap("<html>"
```

Comment:
This is almost certainly not the best way to do this, however it's the best _I_ could come up with.